.github/workflows/main.yml

```diff
@@ -13,15 +13,21 @@ jobs:
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
     steps:
-      - uses: actions/checkout@v4
+      - name: Check out Git repository
+        uses: actions/checkout@v4
 
-      - name: Install uv
-        uses: astral-sh/setup-uv@v4
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Install dependencies
```

Comment:
this could be "install the package"

then you only need to install pytest for the tests.

Reply:
Maybe "install packages"? In case the trick with `uv` will work.

Reply:
Renamed to "install packages" for now.